resources/main.css

```diff
@@ -310,7 +309,17 @@ button.show-about {
     top: 0;
     left: 0;
     height: 6px;
-    width: 0;
+    width: 100%;
+    appearance: none;
+    border: none;
+    background-color: var(--inactive-color);
+}
+
+#progress-completed::-webkit-progress-value {
+    background-color: var(--foreground);
+}
+
+#progress-completed::-moz-progress-bar {
```

Comment:
any way to make the progress smoother? 
maybe something simple like:
`transition: all 0.3s linear;` 🤷

Reply:
why not but it's not 100% needed as usually the steps are short in width, so it's not hugely visible anyway...

Reply:
I think it want to have as little work as possible happening between the benchmark steps.
I'd prefer not having animations (except in workloads maybe).